consensus/src/simplex/config.rs

```diff
@@ -51,6 +51,9 @@ pub struct Config<
     pub relay: R,
 
     /// Reporter for the consensus engine.
+    ///
+    /// Consider wrapping with [`crate::simplex::signing_scheme::reporter::AttributableReporter`]
```

Comment:
nit: I think we could prepend this comment with something along the lines of "all activity is exported for downstream applications that benefit from total observability" or something.